plugin/pkg/auth/authorizer/rbac/bootstrappolicy/namespace_policy.go

```diff
@@ -85,13 +85,6 @@ func init() {
 			rbac.NewRule("get", "list", "watch").Groups(legacyGroup).Resources("secrets").RuleOrDie(),
 		},
 	})
-	addNamespaceRole(metav1.NamespaceSystem, rbac.Role{
-		// role for the cloud providers to access/create kube-system configmaps
-		ObjectMeta: metav1.ObjectMeta{Name: saRolePrefix + "cloud-provider"},
```

Comment:
Just to make sure, is GCE the only consumer of this `cloud-provider` role?

Reply:
I checked kubernetes core and didn't see any other users.

Reply:
I think this may need to follow the [deprecation policy](https://kubernetes.io/docs/reference/deprecation-policy/), probably covered by rule #7:
> Rule #7: Deprecated behaviors must function for no less than 1 year after their announced deprecation.

Has this removal been announced?

Reply:
/cc @kubernetes/sig-auth-pr-reviews